lib/index.ts

```diff
@@ -67,9 +67,9 @@ export function splitBuildStream(
 export async function fromImageDescriptors(
 	images: Compose.ImageDescriptor[],
 	buildStream: Stream.Readable,
-	metadataDirectory: string = '.balena/',
+	metadataDirectories = ['.balena/', '.resin/'],
```

Comment:
`.resin` for compatibility with the older [gist documentation](https://gist.github.com/CameronDiver/6d5964c0618a6154f89d73b883fb09b7) that some customers may still be using in production.

Reply:
Given that we've never officially supported that, we should try to phase it out

Reply:
Agreed, it will be easy to change this in a future PR. The existing code before this fix PR supported `resin.yml`, which I thought "goes together" with a metadata directory named `.resin`.